### PR TITLE
Coastal glider deploy csv review

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: deployment/CP05MOAS-GL389_Deploy.csv [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+TS20190205,,UNRECOVERED,,CP05MOAS-GL389-01-ADCPAM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-ADCPAM-44067,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
+TS20190205,,UNRECOVERED,,CP05MOAS-GL389-02-FLORTM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-FLORTM-04198,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
+TS20190205,,UNRECOVERED,,CP05MOAS-GL389-03-CTDGVM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-CTDGVM-09349,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
+TS20190205,,UNRECOVERED,,CP05MOAS-GL389-04-DOSTAM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-DOSTAM-00189,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
+TS20190205,,UNRECOVERED,,CP05MOAS-GL389-05-PARADM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-PARADM-50222,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
```
Comment (dbogorff) [inline deployment/CP05MOAS-GL389_Deploy.csv:37]: 389 looks good!

File: deployment/CP05MOAS-PG564_Deploy.csv [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+WJ20170815,,TA20171007,,CP05MOAS-PG564-01-CTDGVM000,2,1,2017-08-17T10:20:00,2017-10-07T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-CTDGVM-09286,40.02335,-70.88022,,200,200,CNPM
+WJ20170815,,TA20171007,,CP05MOAS-PG564-02-DOSTAM000,2,1,2017-08-17T10:20:00,2017-10-07T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-DOSTAM-00432,40.02335,-70.88022,,200,200,CNPM
+WJ20170815,,TA20171007,,CP05MOAS-PG564-03-FLORTM000,2,1,2017-08-17T10:20:00,2017-10-07T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTM-03814,40.02335,-70.88022,,200,200,CNPM
+WJ20170815,,TA20171007,,CP05MOAS-PG564-04-FLORTO000,2,1,2017-08-17T10:20:00,2017-10-07T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-FLORTO-01331,40.02335,-70.88022,,200,200,CNPM
+WJ20170815,,TA20171007,,CP05MOAS-PG564-05-NUTNRM000,2,1,2017-08-17T10:20:00,2017-10-07T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-NUTNRM-00657,40.02335,-70.88022,,200,200,CNPM
+WJ20170815,,TA20171007,,CP05MOAS-PG564-06-PARADM000,2,1,2017-08-17T10:20:00,2017-10-07T15:26:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00564,CGINS-PARADM-50199,40.02335,-70.88022,,200,200,CNPM
```
Comment (dbogorff) [inline deployment/CP05MOAS-PG564_Deploy.csv:15]: 564 looks good!

File: deployment/CP05MOAS-PG583_Deploy.csv [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
+SI20170418,,TS20170727,,CP05MOAS-PG583-00-ENG000000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGCON-GLDRGP-00583,40.37043,-70.95436,,200,200,CNPM
+SI20170418,,TS20170727,,CP05MOAS-PG583-01-CTDGVM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-CTDGVM-09307,40.37043,-70.95436,,200,200,CNPM
+SI20170418,,TS20170727,,CP05MOAS-PG583-02-DOSTAM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-DOSTAM-00539,40.37043,-70.95436,,200,200,CNPM
+SI20170418,,TS20170727,,CP05MOAS-PG583-03-FLORTM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTM-04093,40.37043,-70.95436,,200,200,CNPM
+SI20170418,,TS20170727,,CP05MOAS-PG583-04-FLORTO000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-FLORTO-01349,40.37043,-70.95436,,200,200,CNPM
+SI20170418,,TS20170727,,CP05MOAS-PG583-05-NUTNRM000,1,1,2017-04-19T05:30:47,2017-07-27T09:10:00,CGVCP-05MOAS-00000,CGVEH-GLDRGP-00583,CGINS-NUTNRM-00709,40.37043,-70.95436,,200,200,CNPM
```
Comment (dbogorff) [inline deployment/CP05MOAS-PG583_Deploy.csv:7]: 583 looks good!

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
 TS20200301,,,,CP05MOAS-GL379-00-ENG000000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,39.9163,-70.36053,,1000,1000,SS-1
 TS20200301,,,,CP05MOAS-GL379-01-ADCPAM000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-50865,39.9163,-70.36053,,1000,1000,SS-1
 TS20200301,,,,CP05MOAS-GL379-02-FLORTM000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04201,39.9163,-70.36053,,1000,1000,SS-1
 TS20200301,,,,CP05MOAS-GL379-03-CTDGVM000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09347,39.9163,-70.36053,,1000,1000,SS-1
 TS20200301,,,,CP05MOAS-GL379-04-DOSTAM000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,39.9163,-70.36053,,1000,1000,SS-1
-TS20200301,,,,CP05MOAS-GL379-05-PARADM000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,39.9163,-70.36053,,1000,1000,SS-1
\ No newline at end of file
+TS20200301,,,,CP05MOAS-GL379-05-PARADM000,9,1,2020-03-02T09:44:16,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,39.9163,-70.36053,,1000,1000,SS-1
```
Comment (dbogorff) [inline deployment/CP05MOAS-GL379_Deploy.csv:55]: 379 looks good!

File: deployment/CP05MOAS-GL335_Deploy.csv [hunk 5]
```diff
@@ -1,31 +1,31 @@
...
+WJ20170115,,TS20170306,,CP05MOAS-GL335-00-ENG000000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGCON-GLDRCP-00335,40.042333,-70.37853,,200,200,EB
+WJ20170115,,TS20170306,,CP05MOAS-GL335-01-ADCPAM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-ADCPAM-43106,40.042333,-70.37853,,200,200,EB
+WJ20170115,,TS20170306,,CP05MOAS-GL335-02-FLORTM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-FLORTM-04466,40.042333,-70.37853,,200,200,EB
+WJ20170115,,TS20170306,,CP05MOAS-GL335-03-CTDGVM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-CTDGVM-09364,40.042333,-70.37853,,200,200,EB
+WJ20170115,,TS20170306,,CP05MOAS-GL335-04-DOSTAM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-DOSTAM-00310,40.042333,-70.37853,,200,200,EB
+WJ20170115,,TS20170306,,CP05MOAS-GL335-05-PARADM000,5,1,2017-01-16T14:59:00,2017-03-06T22:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00335,CGINS-PARADM-50237,40.042333,-70.37853,,200,200,EB
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL335_Deploy.csv:31]: Ok

Comment (pbrickley) [inline deployment/CP05MOAS-GL335_Deploy.csv:31]: OK

File: deployment/CP05MOAS-GL336_Deploy.csv [hunk 6]
```diff
@@ -1,55 +1,55 @@
...
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-01-ADCPAM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43111,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-02-FLORTM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-03206,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-03-CTDGVM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-04-DOSTAM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL336_Deploy.csv:55]: OK. 
Do we need to have notes also say "unrecovered"?

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 7]
```diff
@@ -1,58 +1,58 @@
...
+TS20181209,,WJ20190228,,CP05MOAS-GL339-03-CTDGVM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09083,40.082183,-70.1885,,200,200,EB
+TS20181209,,WJ20190228,,CP05MOAS-GL339-04-DOSTAM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00566,40.082183,-70.1885,,200,200,EB
+TS20181209,,WJ20190228,,CP05MOAS-GL339-05-PARADM000,9,1,2018-12-09T16:36:49,2019-03-01T14:19:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50203,40.082183,-70.1885,,200,200,EB
 TS20200301,,,,CP05MOAS-GL339-00-ENG000000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGCON-GLDRCP-00339,40.08132,-70.372867,,200,200,EB
 TS20200301,,,,CP05MOAS-GL339-01-ADCPAM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-ADCPAM-72777,40.08132,-70.372867,,200,200,EB
 TS20200301,,,,CP05MOAS-GL339-02-FLORTM000,10,1,2020-03-02T11:09:59,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04467,40.08132,-70.372867,,200,200,EB
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL339_Deploy.csv:58]: OK

File: deployment/CP05MOAS-GL374_Deploy.csv [hunk 8]
```diff
@@ -1,30 +1,31 @@
...
+SI20170418,,UNRECOVERED,,CP05MOAS-GL374-01-ADCPAM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-ADCPAM-49653,40.02882,-70.48575,,200,200,EB;glider lost at sea
+SI20170418,,UNRECOVERED,,CP05MOAS-GL374-02-FLORTM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-FLORTM-04468,40.02882,-70.48575,,200,200,EB;glider lost at sea
+SI20170418,,UNRECOVERED,,CP05MOAS-GL374-03-CTDGVM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-CTDGVM-09361,40.02882,-70.48575,,200,200,EB;glider lost at sea
+SI20170418,,UNRECOVERED,,CP05MOAS-GL374-04-DOSTAM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-DOSTAM-00012,40.02882,-70.48575,,200,200,EB;glider lost at sea
+SI20170418,,UNRECOVERED,,CP05MOAS-GL374-05-PARADM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-PARADM-50240,40.02882,-70.48575,,200,200,EB;glider lost at sea
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL374_Deploy.csv:31]: OK,

same question about notes matching "unrecovered".

File: deployment/CP05MOAS-GL375_Deploy.csv [hunk 9]
```diff
@@ -1,7 +1,7 @@
...
+SI20150110,,UNRECOVERED,,CP05MOAS-GL375-01-ADCPAM000,1,1,2015-01-11T19:47:00,2015-02-13T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00375,CGINS-ADCPAM-49015,40.01988,-70.39292,,1000,1000,FZ-1;glider lost at sea
+SI20150110,,UNRECOVERED,,CP05MOAS-GL375-02-FLORTM000,1,1,2015-01-11T19:47:00,2015-02-13T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00375,CGINS-FLORTM-02817,40.01988,-70.39292,,1000,1000,FZ-1;glider lost at sea
+SI20150110,,UNRECOVERED,,CP05MOAS-GL375-03-CTDGVM000,1,1,2015-01-11T19:47:00,2015-02-13T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00375,CGINS-CTDGVM-09081,40.01988,-70.39292,,1000,1000,FZ-1;glider lost at sea
+SI20150110,,UNRECOVERED,,CP05MOAS-GL375-04-DOSTAM000,1,1,2015-01-11T19:47:00,2015-02-13T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00375,CGINS-DOSTAM-99999,40.01988,-70.39292,,1000,1000,FZ-1;glider lost at sea
+SI20150110,,UNRECOVERED,,CP05MOAS-GL375-05-PARADM000,1,1,2015-01-11T19:47:00,2015-02-13T00:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00375,CGINS-PARADM-50155,40.01988,-70.39292,,1000,1000,FZ-1;glider lost at sea
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL375_Deploy.csv:7]: OK,

File: deployment/CP05MOAS-GL340_Deploy.csv [hunk 10]
```diff
@@ -58,4 +58,4 @@ TS20191212,,AR42-02,,CP05MOAS-GL340-02-FLORTM000,10,1,2019-12-13T01:04:47,2020-0
 TS20191212,,AR42-02,,CP05MOAS-GL340-03-CTDGVM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-CTDGVM-09029,40.0817,-70.74813,,200,200,EB 
 TS20191212,,AR42-02,,CP05MOAS-GL340-04-DOSTAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-DOSTAM-00398,40.0817,-70.74813,,200,200,EB 
 TS20191212,,AR42-02,,CP05MOAS-GL340-05-PARADM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-PARADM-50145,40.0817,-70.74813,,200,200,EB 
-TS20191212,,AR42-02,,CP05MOAS-GL340-01-ADCPAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0817,-70.74813,,200,200,EB 
\ No newline at end of file
+TS20191212,,AR42-02,,CP05MOAS-GL340-01-ADCPAM000,10,1,2019-12-13T01:04:47,2020-03-03T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00340,CGINS-ADCPAM-48878,40.0817,-70.74813,,200,200,EB
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL340_Deploy.csv:61]: ok

File: deployment/CP05MOAS-GL374_Deploy.csv (hunk 8 above)
Comment (pbrickley) [inline deployment/CP05MOAS-GL374_Deploy.csv:31]: ok

File: deployment/CP05MOAS-GL375_Deploy.csv (hunk 9 above)
Comment (pbrickley) [inline deployment/CP05MOAS-GL375_Deploy.csv:7]: ok

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 11]
```diff
@@ -1,64 +1,64 @@
...
+WJ20190617,,MZ20190626,,CP05MOAS-GL376-03-CTDGVM000,10,1,2019-06-18T23:36:35,2019-06-26T18:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09029,39.91363,-70.9107,,1000,1000,FZ-2
+WJ20190617,,MZ20190626,,CP05MOAS-GL376-04-DOSTAM000,10,1,2019-06-18T23:36:35,2019-06-26T18:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00364,39.91363,-70.9107,,1000,1000,FZ-2
+WJ20190617,,MZ20190626,,CP05MOAS-GL376-05-PARADM000,10,1,2019-06-18T23:36:35,2019-06-26T18:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50145,39.91363,-70.9107,,1000,1000,FZ-2
 TS20200301,,,,CP05MOAS-GL376-00-ENG000000,11,1,2020-03-02T09:46:04,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,39.91503,-70.361067,,1000,1000,SS-2
 TS20200301,,,,CP05MOAS-GL376-01-ADCPAM000,11,1,2020-03-02T09:46:04,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49696,39.91503,-70.361067,,1000,1000,SS-2
 TS20200301,,,,CP05MOAS-GL376-02-FLORTM000,11,1,2020-03-02T09:46:04,,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03116,39.91503,-70.361067,,1000,1000,SS-2
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL376_Deploy.csv:64]: ok

File: deployment/CP05MOAS-GL379_Deploy.csv (hunk 4 above)
Comment (pbrickley) [inline deployment/CP05MOAS-GL379_Deploy.csv:55]: ok

File: deployment/CP05MOAS-GL380_Deploy.csv [hunk 12]
```diff
@@ -1,42 +1,42 @@
...
 AR-31,,AR-31,,CP05MOAS-GL380-00-ENG000000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGCON-GLDRCP-00380,39.87505,-70.88473,,1000,1000,SS-1
-AR-31,,AR-31,,CP05MOAS-GL380-01-ADCPAM000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-00545,39.87505,-70.88473,,1000,1000,SS-1
+AR-31,,AR-31,,CP05MOAS-GL380-01-ADCPAM000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-ADCPAM-48878,39.87505,-70.88473,,1000,1000,SS-1
 AR-31,,AR-31,,CP05MOAS-GL380-02-FLORTM000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-FLORTM-05133,39.87505,-70.88473,,1000,1000,SS-1
 AR-31,,AR-31,,CP05MOAS-GL380-03-CTDGVM000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-CTDGVM-09453,39.87505,-70.88473,,1000,1000,SS-1
 AR-31,,AR-31,,CP05MOAS-GL380-04-DOSTAM000,7,1,2018-10-25T20:23:00,2018-10-27T02:47:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00380,CGINS-DOSTAM-00195,39.87505,-70.88473,,1000,1000,SS-1
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL380_Deploy.csv:42]: OK.  End of file line 49, but for some reason I can't add the comment there.

File: deployment/CP05MOAS-GL387_Deploy.csv [hunk 13]
```diff
@@ -1,42 +1,42 @@
...
 AR-34,,WJ20190617,,CP05MOAS-GL387-00-ENG000000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGCON-GLDRCP-00387,39.9681,-71.00002,,1000,1000,FZ-1
-AR-34,,WJ20190617,,CP05MOAS-GL387-01-ADCPAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-00570,39.9681,-71.00002,,1000,1000,FZ-1
+AR-34,,WJ20190617,,CP05MOAS-GL387-01-ADCPAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-ADCPAM-72777,39.9681,-71.00002,,1000,1000,FZ-1
 AR-34,,WJ20190617,,CP05MOAS-GL387-02-FLORTM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-FLORTM-04465,39.9681,-71.00002,,1000,1000,FZ-1
 AR-34,,WJ20190617,,CP05MOAS-GL387-03-CTDGVM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-CTDGVM-09027,39.9681,-71.00002,,1000,1000,FZ-1
 AR-34,,WJ20190617,,CP05MOAS-GL387-04-DOSTAM000,7,1,2019-04-08T01:35:00,2019-06-18T10:34:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00387,CGINS-DOSTAM-00569,39.9681,-71.00002,,1000,1000,FZ-1
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL387_Deploy.csv:42]: Yep

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 14]
```diff
@@ -1,48 +1,48 @@
...
 WJ20190617,,AR-38,,CP05MOAS-GL388-00-ENG000000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.92227,-70.91603,,1000,1000,SS-2
-WJ20190617,,AR-38,,CP05MOAS-GL388-01-ADCPAM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-00537,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,AR-38,,CP05MOAS-GL388-01-ADCPAM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-49016,39.92227,-70.91603,,1000,1000,SS-2
 WJ20190617,,AR-38,,CP05MOAS-GL388-02-FLORTM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03190,39.92227,-70.91603,,1000,1000,SS-2
 WJ20190617,,AR-38,,CP05MOAS-GL388-03-CTDGVM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09083,39.92227,-70.91603,,1000,1000,SS-2
 WJ20190617,,AR-38,,CP05MOAS-GL388-04-DOSTAM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00170,39.92227,-70.91603,,1000,1000,SS-2
```
Comment (pbrickley) [inline deployment/CP05MOAS-GL388_Deploy.csv:48]: EOF Line 55, looks good.

File: deployment/CP05MOAS-GL389_Deploy.csv (hunk 1 above)
Comment (pbrickley) [inline deployment/CP05MOAS-GL389_Deploy.csv:37]: OK, check EOF newline?

File: deployment/CP05MOAS-PG564_Deploy.csv (hunk 2 above)
Comment (pbrickley) [inline deployment/CP05MOAS-PG564_Deploy.csv:15]: Yep

File: deployment/CP05MOAS-PG583_Deploy.csv (hunk 3 above)
Comment (pbrickley) [inline deployment/CP05MOAS-PG583_Deploy.csv:7]: A-OK

**Pull request merged**